src/rules/linebreakStyleRule.ts

```diff
@@ -0,0 +1,61 @@
+import * as ts from "typescript";
+import * as Lint from "../lint";
+
+export class Rule extends Lint.Rules.AbstractRule {
+    public static FAILURE_STRINGS = {
+        CR: "Expected linebreak to be 'CR'",
+        CRLF: "Expected linebreak to be 'CRLF'",
+        LF: "Expected linebreak to be 'LF'",
+    };
+
+    public apply(sourceFile: ts.SourceFile): Lint.RuleFailure[] {
+        const failures = <Lint.RuleFailure[]>[];
+        const scanner = ts.createScanner(
+            sourceFile.languageVersion,
+            false,
+            sourceFile.languageVariant,
+            sourceFile.getFullText()
+        );
+
+        let expectedEOL: string;
+        let failureString: string;
+
+        switch (this.getOptions().ruleArguments[0]) {
+            case "CR":
+                // expectedEOL = "\r";
+                failureString = Rule.FAILURE_STRINGS.CR;
+                break;
+            case "CRLF":
+                expectedEOL = "\r\n";
+                failureString = Rule.FAILURE_STRINGS.CRLF;
+                break;
+            case "LF":
```

Comment:
they do indeed [use LF as the default](http://eslint.org/docs/rules/linebreak-style#options)